projects/rulesets-base/src/extended-rules/spectral-rule.ts

```diff
@@ -145,3 +161,50 @@ export class SpectralRule extends ExternalRuleBase {
     return results;
   }
 }
+
+async function runSpectral(
+  spectralRuleset: string,
+  afterSpecAbsolutePath: string
+): Promise<string> {
+  const resultsOutput = path.join(
+    os.tmpdir(),
+    `spectral-output-${Math.floor(Math.random() * 100000)}.json`
+  );
+
+  return new Promise((resolve, reject) => {
+    const spectralRunning = spawn(
```

Comment:
Spectral has a bug that truncates stdout, they suggest using `--o` and a results file